src/app/registration/park-details/park-details.component.html

```diff
@@ -27,4 +27,22 @@ <h2 class="mb-0">Parking Map <span class="material-icons rotate-45 d-none d-md-i
             </div>
         </div>
     </button>
-</div>
\ No newline at end of file
+</div>
+<div class="mb-3" *ngIf="park?.videoLink" (click)="openMap()">
```

Comment:
Should not call `openMap()` here - should open video.

Reply:
Addressed in new commit. Thanks for review!

Reply:
Fixed in 2e332cdad7f1ed92c0b30de20664aceb66c09c09